### PR TITLE
chore: update node bindings to 1.0.0-alpha.0

### PR DESCRIPTION
**Pull request by ralphtheninja**
Maybe leave this hanging until we have gotten `v0.104.0` out.

Closes https://github.com/deltachat/deltachat-desktop/issues/692
Closes https://github.com/deltachat/deltachat-desktop/issues/841
Closes https://github.com/deltachat/deltachat-desktop/issues/687
Closes https://github.com/deltachat/deltachat-desktop/issues/606

### CODE REVIEW

File: src/renderer/components/Login.js [hunk 1]
```diff
@@ -48,19 +50,18 @@ export default class Login extends React.Component {
...
+      mail_user: this.props.mail_user || '',
+      mail_pw: process.env.DC_MAIL_PW || this.props.mail_pw || '',
+      mail_server: this.props.mail_server || '',
+      mail_port: this.props.mail_port || '',
+      mail_security: this.props.mail_security || '',
+      send_user: this.props.send_user || '',
+      send_pw: this.props.send_pw || '',
+      send_server: this.props.send_server || '',
+      send_port: this.props.send_port || '',
+      send_security: this.props.send_security || ''
```
Comment (ralphtheninja) [inline src/renderer/components/Login.js]: These options matches exactly 1-1 to the ones in core.

**Comment by ralphtheninja** [comment]
Once this has been merged, we should go over the ci again. In theory we should be able to ship windows after this. However, needs testing as well.

**Comment by ralphtheninja** [comment]
Hmm. Maybe we should just merge this into master right now and instead branch off when making the next release (the last using c-core and updated rpgp).

**Comment by Jikstra** [comment]
@ralphtheninja we should first merge https://github.com/deltachat/deltachat-desktop/pull/844 otherwise we need to cherry pick on that other branch. But yes, lets get it merged asap.

File: .travis.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ env:
 
   matrix:
     - DC_CORE_VERSION="master" SYS_DC_CORE="false"
-    - DC_CORE_VERSION="master" SYS_DC_CORE="true"
+#    - DC_CORE_VERSION="master" SYS_DC_CORE="true"
```
Comment (Jikstra) [inline .travis.yml]: Why did you disable this?

Comment (ralphtheninja) [reply .travis.yml]: Because it's not working.

Comment (Jikstra) [reply .travis.yml]: This explains things ^^ It's not very important, we only need it if we want to use a system wide installed libdeltachat. As there's currently no such a package using this, i'm fine with breaking it.

Comment (ralphtheninja) [reply .travis.yml]: We first need to make it happen in rust core, i.e. there's no `make install` step there yet to install the library system wide. So I figured we scrap it in desktop until that is working properly.

Comment (Jikstra) [reply .travis.yml]: Yup i'm fine with this :) Let's merge!

**Pull request merged**